### ISSUE TITLE
Fix app layout issues with larger font sizes

### ISSUE THREAD

**Issue by geosaaga** [fmidev/weather-app#920]
There are various issues when the user has selected larger font size than the default.

**Comment by helmines25** [comment]
According to WCAG guidelines, all text should be resizable up to 200% without loss of content or functionality. The maximum font size settings seem to vary across versions and platforms and determining the exact percentage increase for the largest available sizes can be difficult, as sources differ on this information. The general best practice seems to be to test the app using the highest available font and display size settings on both iOS and Android (this may sometimes mean exceeding the 200% minimum requirement) and ensuring the app works correctly with these settings.
